src/Microsoft.Management.UI.Internal/HelpWindow/ParagraphSearcher.cs

```diff
@@ -43,8 +43,8 @@ internal ParagraphSearcher()
         /// <returns>The next highlight starting at the <paramref name="caretPosition"/>.</returns>
         internal Run MoveAndHighlightNextNextMatch(bool forward, TextPointer caretPosition)
         {
-            Debug.Assert(caretPosition != null, "a caret position is allways valid");
-            Debug.Assert(caretPosition.Parent != null && caretPosition.Parent is Run, "a caret PArent is allways a valid Run");
+            Debug.Assert(caretPosition != null, "a caret position is always valid");
+            Debug.Assert(caretPosition.Parent != null && caretPosition.Parent is Run, "a caret parent is always a valid Run");
```

Comment:
I guess it is reference to property name.
```suggestion
            Debug.Assert(caretPosition.Parent != null && caretPosition.Parent is Run, "a caret Parent is always a valid Run");
```